selection_sort/cpp/Cpp selection_sort.cpp

```diff
@@ -0,0 +1,35 @@
+/* C++ Program - Selection Sort */
+		
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmphixsekeh/selection_sort/cpp/Cpp selection_sort.cpp
+++ b/tmp/tmphixsekeh/selection_sort/cpp/Cpp selection_sort.cpp
@@ -1,5 +1,5 @@
 /* C++ Program - Selection Sort */
-		
+
 #include<iostream.h>
 #include<conio.h>
 void main()
```